pkg/controllers/disruption/suite_test.go

```diff
@@ -121,12 +124,26 @@ var _ = BeforeEach(func() {
 		}
 		return false
 	})
-	// Sort the instances by pricing from low to high
+	// Sort the on-demand instances by pricing from low to high
 	sort.Slice(onDemandInstances, func(i, j int) bool {
 		return onDemandInstances[i].Offerings.Cheapest().Price < onDemandInstances[j].Offerings.Cheapest().Price
 	})
 	leastExpensiveInstance, mostExpensiveInstance = onDemandInstances[0], onDemandInstances[len(onDemandInstances)-1]
 	leastExpensiveOffering, mostExpensiveOffering = leastExpensiveInstance.Offerings[0], mostExpensiveInstance.Offerings[0]
+	spotInstances = lo.Filter(cloudProvider.InstanceTypes, func(i *cloudprovider.InstanceType, _ int) bool {
```

Comment:
I see you're constructing these, but we're not using them in some of the tests. Any reason why? It would be great if we have a set of on-demand instance types, spot instance types, and mixed instance types, and you can do the following at the start of each test so you don't have to re-generate, re-sort, and re-apply. 
```
cloudProvider.InstanceTypes = spotInstances[:15]
node.Labels = propagate_from( spotInstances[:15].Labels )  // <- fake function
```

Reply:
>I see you're constructing these, but we're not using them in some of the tests. Any reason why?
I think I am using spotInstance indirectly though `leastExpensiveSpotInstance` and `mostExpensiveSpotInstance` in almost all the tests until and unless I need to modify it to fit certain corner cases.

>It would be great if we have a set of on-demand instance types, spot instance types, and mixed instance types
I need to check if we have the tests already considering these since we had consolidation already supported for mixture of [OD, Spot] instances right ?